core/src/main/java/edu/wpi/grip/core/util/Holder.java

```diff
@@ -0,0 +1,42 @@
+
+package edu.wpi.grip.core.util;
+
+/**
+ * Simple thread-safe class to allow for setting values from lambdas.
+ */
+public final class Holder<T> {
```

Comment:
It seems like the only usage is in the source.
Why can't you just store the data in a variable.
What advantage does this abstraction give you?